src/core/core/rpc_message.cpp

```diff
@@ -214,6 +214,13 @@ message_ex::message_ex()
 
 message_ex::~message_ex()
 {
+    // when recv a request, message_header is hidden ahead of buffer(see@create_receive_message
```

Comment:
I don't understand what does "hidden ahead of buffer" mean? Could you make it clear? 
BTW, don't use abbreviation like "recv" in your comment. Always be careful of your grammar.